config/initializers/loa.rb

```diff
@@ -8,5 +8,6 @@ module LOA
 
   IDME_LOA1 = 'http://idmanagement.gov/ns/assurance/loa/1/vets'
   IDME_LOA2 = 'http://idmanagement.gov/ns/assurance/loa/2/vets'
-  IDME_LOA3 = 'http://idmanagement.gov/ns/assurance/loa/3/vets'
+  IDME_LOA3 = 'http://idmanagement.gov/ns/assurance/loa/3'
+  IDME_LOA3_VETS = 'http://idmanagement.gov/ns/assurance/loa/3/vets'
```

Comment:
If you're going to change `IDME_LOA3` to `IDME_LOA3_VETS`, then you should probably stay consistent also do it for 1 & 2 as well. Since that will require some annoying updates to code, you might want to save yourself a headache and just call this new one `LOA3`.

Reply:
Actually, I just realized that renaming to LOA3 would result in `LOA::LOA3` 😢 . My main ask is for the `/vets` flavored ones to all be consistent with one another.

Reply:
Yeah...actually not too bad to change the the uses of `LOA1` to `LOA1_VETS` so I'll do that